docs/api/conventions.md

```diff
@@ -91,29 +91,147 @@ There are a few conventions that Remix uses you should be aware of.
+
+#### Layout Routes
+
+```[3-4]
+app
+| routes
+| - blog
+| --- blog.tsx
+| --- categories.tsx
+| --- index.tsx
+| - about.tsx
+| - index.tsx
+| root.tsx
+```
+
+When a nested route has the same name its directory, it becomes a "layout route" for all of the other child routes inside that directory. Similar to your [root route](#root-layout-route), the layout route should render an `<Outlet />` which is where the child routes will appear. This is how you can create multiple levels of persistent layout nesting associated with URLs.
+
+You can also create layout routes **without adding segments to the URL** by prepending the directory and associated route file with `__`. For example, all of your marketing pages could share a layout rendered in `/app/routes/__marketing.tsx` as the layout, and those routes would go in the `/app/routes/__marketing/` directory. A route `/app/routes/__marketing/product.tsx` would be accessible at the `/product` URL.
 
-  Params can be nested routes, just create a folder with the `$` in it.
+#### Dot Delimeters
 
-- **`app/routes/files/$.tsx`**: To add a "splat" path (some people call this a "catchall") name the file simply `$.tsx`. It will create a route path pattern like `files/*`. You can also use this along with dot file names: `app/routes/files.$.tsx`.
+```[8]
+app
+| routes
+| - blog
+| --- blog.tsx
+| --- categories.tsx
+| --- index.tsx
+| - about.tsx
+| - blog.authors.tsx
+| - index.tsx
+| root.tsx
+```
+
+By creating a file with `.` characters between segments, you can create a nested URL without nested layouts. For example, a file `/app/routes/blog.authors.tsx` will route to the pathname `/blog/authors`, but it will not share a layout with routes in the `/app/routes/blog/` directory.
+
+#### Dynamic Route Parameters
+
+```[3]
+app
+| routes
+| - blog
+| --- $postId.tsx
+| --- blog.tsx
+| --- categories.tsx
+| --- index.tsx
+| - about.tsx
+| - blog.authors.tsx
+| - index.tsx
+| root.tsx
+```
+
+Routes that begin with a `$` character indicate the name of a dynamic segment of the URL. It will be parsed and passed to your loader and action data as a value on the `param` object.
+
+For example: `/app/routes/blog/$postId.tsx` will match the following URLs:
+
+- `/blog/my-story`
+- `/blog/once-upon-a-time`
+- `/blog/how-to-ride-a-bike`
+
+On each of these pages, the dynamic segment of the URL path is the value of the parameter in your route's data.
+
+Also noted: nested routes can contain dynamic segments by using the `$` character in the parent's directory name. For example, `/app/routes/blog/$postId/edit.tsx` might represent the editor view for blog entries.
+
+See the <Link to="../routing">routing guide</Link> for more information.
+
+<docs-info>Some CLIs require you to escape the \$ when creating files:
+
+```bash
+touch routes/\$params.tsx
+```
+
+</docs-info>
+
```

Comment:
I don't think enough people will run into this to be worth including in the docs. Not many people use `touch` and those who do will probably know this (or they'll learn pretty quick that it doesn't work and figure it out).

```suggestion
```